src/python/pants/backend/python/subsystems/twine.py

```diff
@@ -5,22 +5,37 @@
 
 from typing import cast
 
+from pants.backend.python.goals.lockfile import PythonLockfileRequest, PythonToolLockfileSentinel
 from pants.backend.python.subsystems.python_tool_base import PythonToolBase
 from pants.backend.python.target_types import ConsoleScript
 from pants.core.util_rules.config_files import ConfigFilesRequest
+from pants.engine.rules import collect_rules, rule
+from pants.engine.unions import UnionRule
 from pants.option.custom_types import file_option, shell_str
+from pants.util.docutil import git_url
 
 
-class TwineSubsystem(PythonToolBase):
+class Twine(PythonToolBase):
```

Comment:
Not a big deal, but why? Imo the old name optimizes for reading rather than writing.

Reply:
That's how the `black`, `isort` etc subsystem classes are named. So it's an attempt to follow the style.

Reply:
Oh yeah I think it'd be better to rename those `BlackSubsystem` etc. Another choice I regret introducing.

But not a big deal, feel free to merge!

Reply:
OK, so let's clean all those up in a follow up :)